src/P3_processes.jl

```diff
@@ -75,25 +76,26 @@ function ice_melt(
     (; K_therm) = aps
     L_f = TD.latent_heat_fusion(tps, Tₐ)
 
-    (; L, N, state) = dist
+    (; L_ice, N_ice) = state
     (; T_freeze, vent) = state.params
 
     v_term = ice_particle_terminal_velocity(state, Chen2022, ρₐ)
     F_v = CO.ventilation_factor(vent, aps, v_term)
+    N′ = N′ice(state, logλ)
 
     # Integrate
     fac = 4 * K_therm / L_f * (Tₐ - T_freeze)
-    dLdt = fac * ∫fdD(dist; ∫kwargs...) do D
-        ∂ice_mass_∂D(state, D) * F_v(D) * N′ice(dist, D) / D
+    dLdt = fac * ∫fdD(state, logλ; ∫kwargs...) do D
+        ∂ice_mass_∂D(state, D) * F_v(D) * N′(D) / D
     end
 
     # only consider melting (not fusion)
     dLdt = max(0, dLdt)
     # compute change of N_ice proportional to change in L
-    dNdt = N / L * dLdt
+    dNdt = N_ice / L_ice * dLdt
 
     # ... and don't exceed the available number and mass of water droplets
-    dNdt = min(dNdt, N / dt)
-    dLdt = min(dLdt, L / dt)
+    dNdt = min(dNdt, N_ice / dt)  # TODO: Apply limiters in CA.jl
```

Comment:
Do you want to keep that limiter here?

Reply:
No, I don't think I really want any limiters in CM, but I'll go through and remove them in a future PR (maybe even the next one!)